server/src/test/java/org/spine3/server/storage/ProjectionStorageShould.java

```diff
@@ -69,6 +85,208 @@ public void return_null_if_no_event_time_in_storage() {
         assertNull(time);
     }
 
+    @SuppressWarnings("MethodWithMultipleLoops")
+    @Test
+    public void read_all_messages() {
+        final int count = 5;
+        final List<I> ids = new LinkedList<>();
+
+        for (int i = 0; i < count; i++) {
+            final I id = newId();
+            final EntityStorageRecord record = EntityStorageRecord.newBuilder()
+                                                                  .setState(Any.getDefaultInstance())
+                                                                  .setWhenModified(Timestamps.getCurrentTime())
+                                                                  .setVersion(1)
+                                                                  .build();
+            storage.write(id, record);
+            ids.add(id);
+        }
+
+        final Map<I, EntityStorageRecord> read = storage.readAll();
+        assertSize(ids.size(), read);
+        for (Map.Entry<I, EntityStorageRecord> record : read.entrySet()) {
+            assertContains(record.getKey(), ids);
+        }
+    }
+
+    @SuppressWarnings("MethodWithMultipleLoops")
+    @Test
+    public void read_all_messages_with_field_mask() {
+        final int count = 5;
+        final List<I> ids = new LinkedList<>();
+
+        final String projectDescriptor = Project.getDescriptor()
+                                                .getFullName();
+        @SuppressWarnings("DuplicateStringLiteralInspection")
```

Comment:
Yes, the inspection is still applicable. The `.id` string is too short to be reported via an inspection, `.name` is not though. Since `name` is a common field name and `.name` is used in other tests we may suppress this inspection.